internal/mode/static/nginx/config/http/config.go

```diff
@@ -12,12 +12,15 @@ type Server struct {
 
 // Location holds all configuration for an HTTP location.
 type Location struct {
-	Return          *Return
-	Rewrites        []string
-	Path            string
-	ProxyPass       string
-	HTTPMatchVar    string
-	ProxySetHeaders []Header
+	Return                *Return
+	Rewrites              []string
+	Path                  string
+	ProxyPass             string
+	HTTPMatchVar          string
+	ProxySetHeaders       []Header
+	AddResponseHeaders    []Header
```

Comment:
Let's introduce a struct for response headers:

```
type ResponseHeaders struct {
    Add []Header
    Set []Header
    Remove []Header
}
```

It reduces the number of fields in Location and I think it's a bit easier to read.

Reply:
Updated https://github.com/nginxinc/nginx-gateway-fabric/pull/1494/commits/629ee127585f277e3e41eafc1d47fd86ea583c43.